src/frontends/paddle/src/op/strided_slice.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <limits.h>
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+Output<Node> idx_node(const std::string& T1, const std::string& T2, const std::string& T3, const NodeContext& node) {
```

Comment:
nit: I don't like `T1`, `T2`, `T3` variable names, could you rename them to something more verbose, e.g. `tensor_alias`, `list_alias`, `attr_alias`

Reply:
renamed